platform/observability/README.md

```diff
@@ -0,0 +1,317 @@
+      "uptime": 3600,
+      "version": "0.0.1",
+      "name": "Archestra Platform API"
+    },
+    "database": {
+      "status": "ok",
+      "message": "Database connection successful"
+    },
+    "providers": [
+      {
+        "provider": "openai",
+        "status": "ok",
+        "message": "OpenAI API key configured"
+      },
+      {
+        "provider": "anthropic",
+        "status": "ok",
+        "message": "Anthropic API key configured"
+      }
+    ]
+  }
+}
+```
+
+### Kubernetes Readiness Check
+```
+GET /ready
+```
+Returns 200 if the service is ready to handle traffic, 503 if not ready. Used by Kubernetes for readiness probes.
+
+### Kubernetes Liveness Check
+```
+GET /live
+```
+Always returns 200 if the process is running. Used by Kubernetes for liveness probes.
+
+## Prometheus Metrics
+
+Archestra exposes Prometheus metrics at the `/metrics` endpoint for comprehensive monitoring.
+
+### Default Metrics
+Archestra automatically collects standard Node.js and process metrics:
+- CPU usage
+- Memory usage
+- Garbage collection statistics
+- Event loop lag
+
+### Application Metrics
+
+#### HTTP Request Metrics
+```
+archestra_http_requests_total{method="POST",status_code="200",route="/v1/openai/chat/completions"}
+archestra_http_request_duration_seconds{method="POST",status_code="200",route="/v1/openai/chat/completions"}
+```
+
+#### LLM Provider Metrics
+```
+archestra_llm_requests_total{provider="openai",model="gpt-4",status="success"}
+archestra_llm_request_duration_seconds{provider="openai",model="gpt-4"}
+```
+
+#### Agent Execution Metrics
+```
+archestra_agent_executions_total{agent_id="abc-123",status="success"}
+archestra_agent_execution_duration_seconds{agent_id="abc-123"}
+```
+
+#### Tool Invocation Metrics
+```
+archestra_tool_invocations_total{tool_name="weather-api",agent_id="abc-123",status="success"}
+archestra_tool_invocation_duration_seconds{tool_name="weather-api",agent_id="abc-123"}
+```
+
+#### MCP Server Metrics
+```
+archestra_mcp_requests_total{server="filesystem",method="read",status="success"}
+archestra_mcp_request_duration_seconds{server="filesystem",method="read"}
+```
+
+#### User Activity Metrics
+```
+archestra_active_users
+archestra_user_sessions_total{status="login"}
+```
+
+#### System Health Metrics
+```
+archestra_system_health_status{component="database"}
+archestra_system_health_status{component="openai"}
+archestra_system_health_status{component="anthropic"}
+```
+
+## Configuration
+
+### Environment Variables
+
+Configure observability through environment variables:
+
+```bash
+# Prometheus metrics
+PROMETHEUS_ENABLED=true
+
+# Health check intervals
+HEALTH_CHECK_INTERVAL=30000
+
+# External service timeouts
+LLM_PROVIDER_TIMEOUT=10000
+DATABASE_TIMEOUT=5000
+```
+
+### Docker Configuration
+
+For Docker deployments, expose the metrics port:
+
+```dockerfile
+EXPOSE 9000  # API port
+EXPOSE 9464  # Metrics port (optional, can use same port)
+```
```

Comment:
sorry but this file is clearly not correct.. there's many points in here that are just not true, for example this portion here.. where does port 9464 come into play? Where/how do the environment variables documented here come into play?